ts/features/it-wallet/components/ItwActionBanner.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+import { View } from "react-native";
+import { Banner } from "../../../components/Banner";
+import { VSpacer } from "../../../components/core/spacer/Spacer";
+import I18n from "../../../i18n";
+
+/**
+ * The base graphical component, take a text as input and dispatch onPress when pressed
+ * include also a close button
+ * @param props
+ * @constructor
+ */
```

Comment:
Maybe we should update this doc by removing @costructor and @param. What do you think?